Sources/Common/Extensions/StringExtensions.swift

```diff
@@ -58,6 +58,10 @@ public extension String {
         return self + characters
     }
 
+    func isBlankOrEmpty() -> Bool {
+        trimmingCharacters(in: .whitespaces).isEmpty
+    }
```

Comment:
I would like some unit tests written to test this function. Even though the function is only 1 line of code, we do need to feel confident that it returns `false` for blank or empty strings.

Reply:
Added test cases.